api_v3/lib/KalturaLiveEntryService.php

```diff
@@ -341,7 +341,12 @@ private function createRecordedEntry(LiveEntry $dbEntry, $mediaServerIndex)
 			$recordedEntry->setModerationStatus($dbEntry->getModerationStatus());
 			$recordedEntry->setIsRecordedEntry(true);
 			$recordedEntry->setTags($dbEntry->getTags());
-			
+
+			// make the recorded entry to be "hidden" in search so it won't return in entry list action
+			if ($dbEntry->getRecordingOptions() && $dbEntry->getRecordingOptions()->getShouldMakeHidden())
```

Comment:
If its nor very complicated think about moving this check to mySearchUtils::setDisplayInSearch which contains the logic of what the displayInSearch value should be for each object